src/app/editor-toolbar/editor-toolbar-save/editor-toolbar-save.component.ts

```diff
@@ -52,7 +54,17 @@ export class EditorToolbarSaveComponent {
           bodyHtml: this.domSanitizer.bypassSecurityTrustHtml('<iframe id="iframe-preview"></iframe>'),
           type: 'confirm',
           onConfirm: () => {
-            this.apiService.saveRecord(this.record);
+            this.apiService.saveRecord(this.record).subscribe(
```

Comment:
Alternative as I said last week,

```
observable.subscribe({
  next: x => console.log('got value ' + x),
  error: err => console.error('something wrong occurred: ' + err),
  complete: () => console.log('done'),
});
```

Instead of relying on the argument order, you can also pass a `{}` with `error`, `next`, `complete` keys.